src/longest_common_subsequence.rs

```diff
@@ -0,0 +1,68 @@
+#![feature(collections)]
+
+// Implements http://rosettacode.org/wiki/Longest_common_subsequence
+
+/// Returns the longest common subsequence of a and b.
+fn longest_common_subsequence(a: &str, b: &str) -> String {
+    let mut lengths = Vec::with_capacity(a.len() + 1);
+    for i in 0..a.len() + 1 {
+        lengths.push(Vec::with_capacity(b.len() + 1));
+        for _ in 0..b.len() + 1 {
+            lengths[i].push(0);
+        }
+    }
```

Comment:
You could remove the for loop and initialize the vec directly using `lengths = vec![vec![0; b.len() + 1]; a.len() + 1];`